com.provar.plugins.provardx/src/commands/provar/metadatacache.ts

```diff
@@ -58,14 +59,34 @@ export default class metadatacache extends SfdxCommand {
         return {};
     }
 
-    this.ux.log("Metadata level" + ' : ' + metadataLevel);
+    this.ux.log("Metadata level" + ' : ' + metadatalevel);
```

Comment:
please revert to camel casing for metadatalevel
Remove log statement for cachePath, PropertFile and metadataLevel